### ISSUE TITLE
email invites: SMTP errors are sticky...

### ISSUE THREAD

**Issue by mherrb** [Ylianst/MeshCentral#1338]
When sending an email invite to install the agent to someone, if there is an error in the SMTP transaction, it seems to "stick" and block any further e-mail from being sent.
After I mistyped an address, I get this in the system log:
```
May 12 17:33:21 reykjavik node[11537]: SMTP server failed: {"code":"EENVELOPE","response":"550 5.1.1 <toto@laas.fr>... User unknown","responseCode":550,"command":"RCPT TO","rejected":["toto@laas.fr"],"rejectedErrors":[{"code":"EENVELOPE","response":"550 5.1.1 <toto@laas.fr>... User unknown","responseCode":550,"command":"RCPT TO","recipient":"toto@laas.fr"}]}
```
and after this, trying to send the invite to the correct email, the same error is logged again, and no mail is sent.

I'm running meshcentral version 0.5.31

**Comment by Ylianst** [comment]
That is not good. Thanks for the report. I will look into this.

**Comment by Ylianst** [comment]
Just published MeshCentral v0.5.32 with improvements in this area. Let me know if it work. MeshCentral will try to send an email 3 times at 10 second interval before giving up and moving on to the next one.

**Comment by mherrb** [comment]
Hi, I've tried v0.5.32, the issue is fixed. I've verified that after an error, new mails get sent.
Thanks a lot for the reactivity.
